provision/internal/operator/terraform/gardener.go

```diff
@@ -26,9 +26,19 @@ func initGardenerProvider() error {
 		return err
 	}
 	providerPath := filepath.Join(pluginDirs[1], providerName)
+	windowsProviderPath := providerPath + ".exe"
 
 	// check if plugin is in the plugins dir
 	if _, err := os.Stat(providerPath); !os.IsNotExist(err) {
+		if runtime.GOOS == "windows" {
```

Comment:
This implementation is only for the gardener terraform provider, what about any other?

Reply:
Hi, the issue where an exe file is not downloaded and a binary is downloaded instead exists only for gardener. Both azure and gcp download exe files directly, and provisioning is successful.